dash/dependencies.py

```diff
@@ -35,6 +35,11 @@ class State(DashDependency):  # pylint: disable=too-few-public-methods
 class ClientsideFunction:
     # pylint: disable=too-few-public-methods
     def __init__(self, namespace=None, function_name=None):
+
+        if namespace in ['PreventUpdate', 'no_update']:
+            raise ValueError('"{}" is a forbidden namespace in'
+                             ' dash_clientside.'.format(namespace))
```

Comment:
FYI @rpkyle - I think this namespace exclusion is the only piece of this PR that's relevant to R